tests/wpt/metadata/html/semantics/embedded-content/media-elements/readyState_during_canplaythrough.html.ini

```diff
@@ -0,0 +1,2 @@
+[readyState_during_canplaythrough.html]
+  expected: TIMEOUT
```

Comment:
ah, I commented on the layout-2020 one.... this should be removed as well 😂

Reply:
Ha, whoops.